networkit/cpp/distance/test/DynSSSPGTest.cpp

```diff
@@ -291,4 +472,39 @@ TEST_F(DynSSSPGTest, testDynamicDijkstraBatches) {
     });
 }
 
+TEST_F(DynSSSPGTest, testDynamicDijkstraBatchesEdgeDeletions) {
+    ErdosRenyiGenerator generator(100, 0.25, false, false);
+    Graph G1 = generator.generate();
+    Graph G(G1, true, false);
+    DEBUG("Generated graph of dimension ", G.upperNodeIdBound());
+    // delete random edges from G
+
+    G.forNodes([&] (node source) {
+        DynDijkstra dyn_dij(G, source, true);
+        Dijkstra dij(G, source);
+        dyn_dij.run();
+        dij.run();
+        DEBUG("Before the edge deletion: ");
+        count batchSize = 8;
+            std::vector<GraphEvent> batch;
+            for (count i = 0; i < batchSize; ++i) {
```

Comment:
```suggestion
            std::vector<GraphEvent> batch;
            for (count i = 0; i < 8; ++i) {
```
Also, you can remove the additional indentation.